docs/docs/05-integrations/02-command-line.md

```
@@ -108,6 +110,28 @@ karakeep --api-key mysupersecretkey --server-addr https://try.karakeep.app whoam
+  "apiKey": "mysupersecretkey"
+}
+```
+
+Command-line options take precedence over environment variables, and
+environment variables take precedence over the config file.
+If no server address is provided, the CLI defaults to
+`https://cloud.karakeep.app`.
+
+To create or update this file interactively, run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)